README.md

```diff
@@ -1269,20 +1290,39 @@ The following sections show the available operations for the Configuration API.
 
 For every Device Group, the pair (resource, apikey) *must* be unique (as it is used to identify which group to assign to which device). Those operations of the API targeting specific resources will need the use of the `resource` and `apikey` parameters to select the apropriate instance.
 
+Note that there is a 1:1 correspondence between payload fields and DB fields (but with some changes in the attribute
+naming; e.g.: subservice -> service_path).
+
 #### Device Group Model
 Device groups contain the following attributes:
-* **service**: service of the devices of this type.
-* **subservice**: subservice of the devices of this type.
-* **resource**: string representing the Southbound resource that will be used to assign a type to a device (e.g.: pathname in the southbound port).
-* **apikey**: API Key string.
-* **type**: name of the type to assign to the group.
-* **trust**: trust token to use for secured access to the Context Broker for this type of devices (optional; only needed for secured scenarios).
-* **cbHost**: Context Broker connection information. This options can be used to override the global ones for specific types of devices.
-* **lazy**: list of lazy attributes of the device. For each attribute, its `name` and `type` must be provided.
-* **commands**: list of commands attributes of the device. For each attribute, its `name` and `type` must be provided.
-* **active**: list of active attributes of the device. For each attribute, its `name` and `type` must be provided.
-* **staticAttributes**: this attributes will be added to all the entities of this group 'as is'.
-* **internalAttributes**: optional section with free format, to allow specific IoT Agents to store information along with the devices in the Device Registry.
+* **service**: .
+* **subservice**:
+* **resource**:
+* **apikey**:
+* **type**:
+* **trust**:
+* **cbHost**:
+* **lazy**:
+* **commands**:
+* **active**:
+* **staticAttributes**:
+* **internalAttributes**:
+
+| Attribute | DB Field | Definition                                     |
```

Comment:
I'd suggest to use "Payload field" (or some alternative not using "attribute") to avoid confussion.